views/partials/what_to_do_svg.hbs

```diff
@@ -0,0 +1,19 @@
+<svg role="img" width="42px" height="42px" viewBox="0 0 42 42" aria-labelledby="title" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>{{tip_title}}</title>
```

Comment:
By some miracle or magic it passes itself from one template to the next.

Reply:
Partials inherit the context from which they are included. When this is included in the `{{#each passwordTips }}` block of `tips.hbs`, it inherits the individual tip context, so it can render its properties.